src/util/url.ts

```diff
@@ -12,3 +12,5 @@ export const getParams = (url: string): any =>  {
 }
 
 export const scriptTypes = ["account", "tx", "script"]
+
+export const localProjectId = "local-project"
```

Comment:
Can we use to all caps with underscore for const name (not the value), i.e. `LOCAL_PROJECT_ID`?
This way it should be easier to catch up that value is constant and defined elsewhere.

Reply:
Got it. This actually crossed my mind while making the changes but for some reason I didn't do it. Change made and pushed to this same branch/PR!